hibikase/LineTimingDecorations.cpp

```diff
@@ -65,6 +61,33 @@ SyllableDecorations::SyllableDecorations(const QPlainTextEdit* text_edit, int st
     setVisible(true);
 }
 
+QColor SyllableDecorations::GetPlayingColor() const
+{
+    return QPalette().color(QPalette::Text);
+}
+
+QColor SyllableDecorations::GetNotPlayedColor() const
+{
+    QColor col(0x55, 0x77, 0x55);
+    // If the system text colour is darker than our colours, make them lighter
```

Comment:
Isn't this actually checking if the system text color is lighter than our colors?

Reply:
Uh… yes :p